pipelines/folioRancher/folioScheduledTesting/runNightlyCypressBugfestTests/Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+#!groovy
+package folioRancher.folioScheduledTesting.runNightlyCypressBugfestTests
+
+
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('pipelines-shared-library@RANCHER-1052') _
```

Comment:
Do not forget to change the branch to RANCHER-741-Jenkins-Enhancements before merge